resource/abstractions/individual_access.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Individual Access
+
+General Resource Profile Administration functions for checking an individual user's access to a General Resource Profile. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_my_access()`
+
+```python
+def get_my_access(self, resource: str, class_name: str) -> Union[str, bytes, None]:
+```
+
+#### 📄 Description
+
+Check **Your Access** for a general resource profile.
```

Comment:
```markdown
Check the **Running User's** access to a general resource profile.
```

Reply:
True, that's a very good distinction to make. Change made.